distribution/resources/bin/gateway

```diff
@@ -167,7 +175,7 @@ java_cmd_gateway () {
     $JAVA_OPTS \
     -Dmgw-runtime.home=${GW_HOME} \
     -Dballerina.home=${GW_HOME} \
-    -jar $args --api.usage.data.path=$GW_HOME/api-usage-data --b7a.http.accesslog.path=$GW_HOME/logs/access_logs --b7a.config.file=$GW_HOME/conf/micro-gw.conf 2>&1 | tee -a $GW_HOME/logs/microgateway.log
+    -jar ${args[0]} --api.usage.data.path=$GW_HOME/api-usage-data --b7a.http.accesslog.path=$GW_HOME/logs/access_logs --b7a.config.file=$GW_HOME/conf/micro-gw.conf ${args[@]:1}  2>&1 | tee -a $GW_HOME/logs/microgateway.log
```

Comment:
As per the discussion we had regarding docker config loading, we don't need this change anymore

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/966/commits/fe1f68c2c9d1d39e961d2664256249498d69c5aa